aesara/tensor/slinalg.py

```diff
@@ -187,6 +187,70 @@ def perform(self, node, inputs, outputs):
     def infer_shape(self, fgraph, node, shapes):
         return [shapes[0]]
 
+class CholeskySolve(Op):
+    """Solve the linear equations A x = b, given the Cholesky factorization of A.
+    Parameters
+    ----------
+    lower : bool:
+        Whether c is lower triangle or upper triangle
+    check_finite : bool, optional
+        Whether to check that the input matrices contain only finite numbers.
+        Disabling may give a performance gain, but may result in problems
+        (crashes, non-termination) if the inputs do contain infinities or NaNs.
+
+    __props__ = (
+
+        "lower",
+        "check_finite",
+    )
+    """
+
+    def __init__(
+        self,
+        lower=True,
+        check_finite=True,
+    ):
+        self.lower = lower
+        self.check_finite = check_finite
+
+    def __repr__(self):
+        return "CholeskySolve{%s}" % str(self._props())
+
+    def make_node(self, C, b):
+        C = as_tensor_variable(C)
+        b = as_tensor_variable(b)
+        assert C.ndim == 2
+        assert b.ndim in [1, 2]
+
+        # infer dtype by solving the most simple
+        # case with (1, 1) matrices
+        o_dtype = scipy.linalg.solve(
+            np.eye(1).astype(C.dtype), np.eye(1).astype(b.dtype)
+        ).dtype
+        x = tensor(broadcastable=b.broadcastable, dtype=o_dtype)
+        return Apply(self, [C, b], [x])
+
+    def perform(self, node, inputs, output_storage):
+        C, b = inputs
+        rval = scipy.linalg.cho_solve(
+                (C, self.lower),b,
```

Comment:
```suggestion
                (C, self.lower), b,
```

You might need to set up `pre-commit` in your repo, especially if it didn't catch this.